reference/implementing-a-concept-exercise.md

```diff
@@ -21,31 +21,30 @@ The directory structure of a single concept exercise looks like this:
 languages
 └── haskell
     ├── concepts
-    |   └── <CONCEPT_SLUG>
+    |   └── <CONCEPT-SLUG>
     |       ├── about.md
+    |       ├── introduction.md
     |       └── links.json
     └── exercises
-        └── concept
-            └── <SLUG>
+        └── concepts
```

Comment:
this should still be concept, since the directory holds concept exercises, not concepts